### DISCUSSION TITLE
Repeatable icons display incorrectly in iOS

### DISCUSSION THREAD

**Discussion by semiplay** [Laravel-Backpack/community-forum#929] [Q&A (Help)]
Had this question in here 2 or 3 weeks ago and it got deleted.  Let's try again.  
Icons in iOS (iphone/ipad, iOS16.x and above tested) don't appear correctly in repeatable steps.  The close button is off center and the up/down arrows don't show at all.  Desktop is fine.  

I've hacked a way to put images in created steps, but the dynamic creation of steps in javascript still shows them incorrectly.  I'd like to know either how this can be fixed the correct way, or where the javascript code is so I can try to override it. Thanks!
![ios](https://github.com/Laravel-Backpack/community-forum/assets/4955961/f0ebbcef-5ab7-41b7-adcb-67bb714377ad)

**Comment by pxpm** [comment]
Hello @semiplay 

We don't delete discussions. I never recall deleting a discussion topic unless I created it by mistake. 👍 

What happened is that we moved the discussion to an issue in the appropriate repository, in this case it's private (it's in PRO), we shouldn't move issues to private repositories where they are not traceable by the users. We will pay more attention to that.

I've just asked my colleague who picked up this issue about more info. 

Sorry the bad experience 🙏

**Reply by semiplay** [reply]
Ah!  My bad, I just couldn't find it  when searching and thought it had been deleted.  Thanks for the update!

**Reply by pxpm** [reply]
You did well 🙏 . Thanks for the heads up, we will work on it in the following days and get back here with updates. 🙏

**Reply by semiplay** [reply]
Hi @pxpm - any updates on this one?

**Comment by semiplay** [comment]
Any word on this one?  Thanks!

**Reply by pxpm** [reply]
Sorry @semiplay no news yet. Thanks for the heads up. 🙏 
I've just pinged my colleges that use iOS/macOS so that they don't let this issue fell through the cracks and can prioritize it. 👍 

Cheers
